keyboards/binepad/bn003/keymaps/via/keymap.c

```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_0,
+        KC_1,
+        KC_2
+    ),
+    [1] = LAYOUT(
+        KC_NO,
+        KC_NO,
+        KC_NO
+    ),
+    [2] = LAYOUT(
+        KC_NO,
+        KC_NO,
+        KC_NO
+    )
```

Comment:
```suggestion
    ),
    [3] = LAYOUT(
        KC_NO,
        KC_NO,
        KC_NO
    )
```
VIA uses 4 layers by default.
Since it is not overwritten here and there is no reason to do so, add another empty layer.